portsyncd/portsyncd.cpp

```diff
@@ -247,54 +242,7 @@ void handlePortConfig(ProducerStateTable &p, map<string, KeyOpFieldsValuesTuple>
             /* No support for port delete yet */
             if (op == SET_COMMAND)
             {
-                
-                for (auto i : values)
```

Comment:
This change is not relevant to the PR description. Please revert

Reply:
This change is related to description:

2. Remove previous workaround https://github.com/Azure/sonic-swss/pull/1769

Reply:
@prsunny we discussed this offline which was a hacky code put earlier as mentioned by junchao. so this code can be removed